src/main/java/org/dswarm/graph/index/Statics.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * This file is part of d:swarm graph extension.
+ *
+ * d:swarm graph extension is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * d:swarm graph extension is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with d:swarm graph extension.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.dswarm.graph.index;
+
+import java.io.File;
+
+/**
+ * @author tgaengler
+ */
+public final class Statics {
+
+	public static final String INDEX_FILE_ENDING = ".dat";
+
+	public static final String USER_DIR_IDENTIFIER = "user.dir";
+
+	public static final String USER_DIR = System.getProperty(USER_DIR_IDENTIFIER);
```

Comment:
No, I think that user.dir is a system property. So it is already set. However, I'm unsure, whether we really use it at the moment. I think that we we probably can remove the DEFAULT_STORE_DIR stuff, since we are able to place the mapdb indices at an appropriate place below neo4j graph.db dir.